src/main/webapp/popular.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Popular Videos</title>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body onload="loadPopular()">
+  <div id="content">
+    <div id="popular-list-container"></div>
+  </div>
+
+  <script src="https://apis.google.com/js/api.js"></script>
+  <script src="scripts/script.js"></script>
+  <script src="scripts/popular.js"></script>
+  <script src="scripts/list-video-service.js"></script>
```

Comment:
When I see multiple scripts on the DOM, I think they are strictly independent. One script might be updating the popular section of the page, another script might be loading ads, another script might loading comments, etc. In this case they are independent and the order they are added is also important. Is it possible to do the import/export we talked about?